my.css

```diff
@@ -352,3 +352,162 @@ body {
     justify-content: center;
   }
 }
+
+/* Вторая секция */
+.bagscluth__content {
+  display: flex;
+  
+}
+.bagscluth {
+  margin: 0;
+  padding: 0;
+  position: relative;
+  max-height: 900px;
+  display: flex;
+  background-color: #fae8e4;
+}
+.bagscluth__img {
+  width: 55%;
+  height: 844px;
+  
+  display: flex;
+  align-items: center;
+}
+.bags-img {
+  position: absolute;
+  display: block;
+  width: 340px;
+  height: 370px;
+  padding-left: 90px;
+
+}
+.bags-img1 {
+  position: absolute;
+  width: 540px;
+  height: 740px;
+  left: 160px;
+}
+
+.inner {
+  width: 400px;
+  position: relative;
+  right: 10px;
+  padding: 50px;
+
+}
+.card {
+  
+  width: 350px;
+  height: 456px;
+  padding: 40px 55px 45px 65px; 
+  margin: 0 auto;
+  top: 200px;
+  display: flex;
+  flex-flow: column;
+  background-color: #fff;
+  border: 1px solid #fff;
+  box-shadow: 29px 30px 22px 0px rgba(0,0,0,0.07);
+
+}
+
+
+.card__text {
+ padding-top: 15px;
+ padding-bottom: 25px;
+ margin: 0 auto;
+}
+.text {
+  opacity: 0.5;
+}
+.card__row {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  padding-bottom: 25px;
+}
+.card-clath-color {
+  width: 180px;
+  height: 40px;
+}
+.card-clath-color__button {
+  width: 30px;
+  height: 30px;
+  border-radius: 100%;
+}
+
+/* цвета */
+
+.card-clath-color__button_color_gray {
+  background-color: #dedede;
+}
+.card-clath-color__button_color_red{
+  background-color: #f29f9f;
+}
+.card-clath-color__button_color_blue{
+  background-color: #a8bff9;
+}
+.card-clath-color__button_color_green{
+  background-color: #90bd8f;
+}
+
+/* Карточка цен */
+
+.choice__row {
+  position: relative;
+  width: 380px;
+  height: 45px;
+}
+.card__choice-product {
+  position: absolute;
+  text-align: center;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 80px;
+  height: 40px;
+  left: 20px;
+  border: 1.5px solid #000000;
+  box-sizing: border-box;
+}
+.choice-product__buton {
+  background-color: #fff;
+  border: #fff;
+  padding: 10px;
+}
+
+.card-basket{
+ position: absolute;
+ left: 115px;
+ width: 255px;
+ height: 40px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/itog/pullRequest?prid=5642160)